src/modules/dashboard/sagas/colonyCreate.ts

```diff
@@ -194,21 +190,6 @@ function* colonyCreate({
       });
     }
 
-    yield createGroupedTransaction(deployOldRoles, {
```

Comment:
Have you tested creating a colony after removing these? I think it will be broken now, since `setOneTxRole` uses `setAdminRole` which is dependent on `OldRoles` being deployed for the colony.

We'll need to replace that one transaction with two different ones: `setAdministrationRole` and `setFundingRole`, both with `domainId: 1`. Not ideal that we will have multiple transactions for this, but we're still overall reducing the number of transactions.

Reply:
Yes, as I wrote in the PR description I tested after making changes in the PR and it worked well, but maybe it would lead to problems later, so I'll see how I get these two things in.

Reply:
Ok, now I see what went wrong, reading this comment I thought you wanted me to remove `setOneTxRole` and replace it. I'll bring it back now.